main.js

```diff
@@ -18,8 +18,10 @@
  *        MIT License. All code unless otherwise specified is
  *        Copyright (c) Matthew Knox and Contributors 2015.
  */
-
+'use strict';
 // Load NodeJS Modifications/Variables
+require('babel-register');
+require('babel-polyfill');
```

Comment:
Ok that's fine as long as its accepted that ES6 code can not be used until prototype.js has been loaded